### PR TITLE
Roadmap update (Aug 2022)

### PR DESCRIPTION
**Pull request by jtcohen6**
Roadmap update, as a markdown file in `docs/roadmap`, three months after the first one

- I'm open to changing the title
- It's 9 pm (CEST) on August 31... still counts

### CODE REVIEW

**Comment by jtcohen6** [comment]
@cla-bot check

(one of us, one of us)

**Comment by lostmygithubaccount** [comment]
btw we/somebody should setup actions so they don't run (or most don't) on these non-code directories

File: docs/roadmap/2022-08-back-for-more.md [hunk 1]
```diff
@@ -86,6 +86,7 @@ If you've been following our GitHub discussions, or the Analytics Engineering ro
...
+3. **Next steps for Python.** Supporting data science workflows, machine learning model trianing, and "MLOps"? Adding more adapaters and Python runtimes? Enbabling writing docs and tests directly in `.py` files? There's a lot to consider here, and we need your help! We've started [#5752](https://github.com/dbt-labs/dbt-core/discussions/5742) to kick off the discussion.
```
Comment (jtcohen6) [inline docs/roadmap/2022-08-back-for-more.md:89]: I moved this further down, under "existing capabilities" - but it's an arbitrary distinction. Happy to have it here instead. Let's just pick one or the other

File: docs/roadmap/2022-08-back-for-more.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+2. **External orchestration.** The same dbt DAG, playing a more active role. We've been developing this idea internally, and have arrived at a few strong opinions. This would not a new node type, but an upgrade to the ones we already have: sources, models, and exposures. Sources that can trigger their own ingest. Exposures that can trigger downstream consumers (syncs, sinks, etc). Models that can define and run transformations in companion execution environments, that read from and write back to the centralized data warehouse. For each external integration, a simple request where it possible, and a dedicated plugin where justified. If you're someone who followed along the original "external nodes" discussion ([#5073](https://github.com/dbt-labs/dbt-core/discussions/5073))—especially if you've got a tool you'd be excited to integrate into dbt's DAG—let's talk.
```
Comment (Fleid) [inline docs/roadmap/2022-08-back-for-more.md:88]: Typo : 'This would not a new node type' >> 'This would not be a new node type' I suppose?

**Review by Fleid — APPROVED** [review]

**Review by gshank — APPROVED** [review]

File: docs/roadmap/2022-08-back-for-more.md (hunk 2 above)
Comment (jtcohen6) [reply docs/roadmap/2022-08-back-for-more.md:88]: tu supposes bien

**Pull request merged**